src/icons/style.css

```diff
@@ -1,10 +1,6 @@
 @font-face {
   font-family: 'icomoon';
-  src:
-    url('fonts/icomoon.woff2?ookznz') format('woff2'),
-    url('fonts/icomoon.ttf?ookznz') format('truetype'),
-    url('fonts/icomoon.woff?ookznz') format('woff'),
-    url('fonts/icomoon.svg?ookznz#icomoon') format('svg');
+  src: url('fonts/icomoon.woff?ookznz') format('woff');
```

Comment:
Because we're inlining this asset as base64, I removed the other fonts from the source stack to use the widely compatible `woff` format.

This helps us keep the CSS from getting huge for now.